### ISSUE TITLE
Radios send duplicate messages when there's more than one source

### ISSUE THREAD

**Issue by JohnFulpWillard** [tgstation/tgstation#79550]
## Reproduction:

Take a bunch of radios, put them all to listen and broadcast, then say something. It will say your message several times over the radio.
This is incredibly easy to exploit and lag the server, especially when paired with tape recorders and/or holopads (not that they should be able to broadcast over radio anyways, but this is still something that needs addressing regardless).

**Comment by BlueMemesauce** [comment]
Isn't that how radios are supposed to work? Each radio receives the message, and then sends it to tcomms. How else would it work?

**Comment by JohnFulpWillard** [comment]
Yeah, any exact replicas (same message, same radio frequency, etc) should be ignored i think.

**Comment by deathrobotpunch** [comment]
video for reference https://youtu.be/t-7mQhSZRgM?si=swEcogEif85pmB1o

**Comment by timothymtorres** [comment]
Duplicate of #59522

**Comment by BlueMemesauce** [comment]
It's not a duplicate, that one is about holodisks being able to play over the radio
this is about putting two radios next to each other and using them both at the same time

**Comment by JohnFulpWillard** [comment]
^
